docs/NextJs.md

```diff
@@ -77,7 +79,7 @@ export default AdminApp;
 
 This is a minimal configuration to render CRUD pages for users, posts and comments. React-admin will guess the fields to display in the list and edition pages based on the API response. 
 
-**Tips**: If you choose App Router, do not forget to add [the `"use client"` directive](https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive).
+**Tips**: If you choose Pages Router, do not forget to remove [the `"use client"` directive](https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive).
```

Comment:
Do we need this tip? If they use the Pages router, the `"use client"` directive does nothing.